test/internal/cmd/generate-gateway-api-urls/main.go

```diff
@@ -12,26 +12,29 @@ import (
 	"text/template"
 )
 
-//go:generate go run . -crds-url $CRDS_URL -raw-repo-url $RAW_REPO_URL -in $INPUT -out $OUTPUT
+//go:generate go run . -standard-crds-url $STANDARD_CRDS_URL -crds-url $CRDS_URL -raw-repo-url $RAW_REPO_URL -in $INPUT -out $OUTPUT
 
 var (
-	crdsURLFlag    = flag.String("crds-url", "", "The URL of Gateway API CRDs to be consumed by kustomize")
-	rawRepoURLFlag = flag.String("raw-repo-url", "", "The raw URL of Gateway API repository")
-	inFlag         = flag.String("in", "", "Template file path")
-	outFlag        = flag.String("out", "", "Output file path where the generate file will be placed")
+	standardCrdsURLFlag = flag.String("standard-crds-url", "", "The URL of standard Gateway API CRDs to be consumed by kustomize")
+	crdsURLFlag         = flag.String("crds-url", "", "The URL of Gateway API CRDs to be consumed by kustomize")
```

Comment:
```suggestion
	crdsStandardURLFlag = flag.String("crds-standard-url", "", "The URL of standard Gateway API CRDs to be consumed by kustomize")
	crdsExperimentalURLFlag         = flag.String("crds-experimental-url", "", "The URL of experimental Gateway API CRDs to be consumed by kustomize")
```

Reply:
WDYT here to change the order of `crd` and `standard` in flags? Is there any standards/recommendations for arranging words in names/flags/env vars?

Reply:
Yeah... this I don't know but as noted in the other comment I'd swap it so that they have `CRDS` prefix so that they are easier to group by and reason about.

Reply:
updated.